src/rt-api/post-contact.ts

```diff
@@ -90,42 +119,78 @@ export const postContact = async (
     });
   }
 
-  if (contact.contactFieldsToDelete){
-    console.log(contact.contactFieldsToDelete)
-    if(contact.contactFieldsToDelete.emailAddresses){
-      for(const internetRecId of contact.contactFieldsToDelete.emailAddresses){
-        const internetResult = await Axios({
-          method: "post",
+  // If present, update contact's url addresses
+  if (contact.Urls) {
+    contact.Urls.forEach(async (url: UrlUpdate) => {
```

Comment:
I would avoid forEach(), it's an outdated style of javascript coding. A for loop `for (const url or contact.Urls)` or a map would be better